presto-hive/src/main/java/com/facebook/presto/hive/s3/HiveS3Config.java

```diff
@@ -56,6 +56,7 @@
     private DataSize s3MultipartMinPartSize = new DataSize(5, MEGABYTE);
     private boolean pinS3ClientToCurrentRegion;
     private String s3UserAgentPrefix = "";
+    private PrestoS3AclType s3AclType = PrestoS3AclType.Private;
```

Comment:
Is `Private` also the default acl that aws sdk uses when we don't specify it? We should confirm that before making this the default.

Reply:
According to https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/s3/model/CannedAccessControlList.html: 

public static final CannedAccessControlList Private
Specifies the owner is granted Permission.FullControl. No one else has access rights.
This is the default access control policy for any new buckets or objects.